scene/3d/mesh_instance.h

```diff
@@ -47,6 +49,31 @@ class MeshInstance : public GeometryInstance {
 	Ref<SkinReference> skin_ref;
 	NodePath skeleton_path;
 
+	struct S_SoftwareSkinning {
```

Comment:
What does the "S_" signify if you dont mind me asking?

Reply:
Let me change that. Sorry for the confusion, it's from a coding standard I was used to in the past, C_ for class and S_ for struct. Old habits die hard :)